requirements.txt

```diff
@@ -0,0 +1 @@
+uncompyle6
```

Comment:
the requirements.txt would need to include `jinja2`.

But this is unrelated to Python 3 support.